packages/expo-image/android/src/main/java/expo/modules/image/GlideModelProvider.kt

```diff
@@ -0,0 +1,79 @@
+package expo.modules.image
```

Comment:
wouldn't it make sense to split into separate files?

Reply:
The `GlideModelProvider` is a sealed class. It's possible to split it between files right now, but I prefer leaving it in one place.